fenv_lmi.hpp

```diff
@@ -121,8 +101,10 @@ enum enum_fenv_indulgence
 
 void LMI_SO fenv_initialize();
 
+#if defined LMI_X87
```

Comment:
Another alternative could be to provide these functions but always return the fixed `fe_dblprec` value from the accessor and throw from the setter. This didn't seem to be very useful to me, but please let me know if I was wrong.